content/ical.md

```diff
@@ -0,0 +1,3 @@
+---
+template: calendar.html
+---
```

Comment:
is this an empty event? Or just some sort of template? How would you subscribe to an event?